### ISSUE TITLE
Secret Agent, Secret Service as an ADJ with a Number?

### ISSUE THREAD

**Issue by AngledLuffa** [UniversalDependencies/UD_English-GUM#78]
There are a couple times where `Secret Service` or `Secret Agent` have `Secret` tagged as UPOS `ADJ` and XPOS `NNP`.  It has the features `Degree=Pos|Number=Sing` in that case.  Perhaps that should just be `Degree=Pos`

```
# sent_id = GUM_interview_mcguire-6
# text = I launched LibriVox, emailed some friends and some podcasters who where doing literary stuff, and invited them to record a chapter of Joseph Conrad's Secret Agent.
28      Secret  Secret  ADJ     NNP     Degree=Pos|Number=Sing  29      amod    29:amod XML=<hi rend:::"italic">
29      Agent   Agent   PROPN   NNP     Number=Sing     23      nmod    23:nmod:of      Entity=27)26)|SpaceAfter=No|XML=</hi>

# sent_id = GUM_speech_maiden-12
# text = The presence of thousands of National Guardsmen, police, and Secret Service was a reminder that 2 weeks ago, in this very room and on those very steps ...
12      Secret  Secret  ADJ     NNP     Degree=Pos|Number=Sing  13      amod    13:amod Entity=(38-organization-new-cf9-1,2-sgl-United_States_Secret_Service
13      Service Service PROPN   NNP     Number=Sing     7       conj    4:nmod:of|7:conj:and    Entity=38)35)
```

related to https://github.com/UniversalDependencies/UD_English-PUD/issues/37

**Comment by nschneid** [comment]
Agreed, `Number` should be restricted to non-ADJ NNPs.

**Comment by rhdunn** [comment]
What about plural forms like "Attorneys General"? Are those "Attorneys/PROPN+NNPS General/ADJ+NNP"? -- Note the switched noun/adj positions. In which case the ADJ+NNP should never be plural.

**Comment by nschneid** [comment]
I think modifiers within a proper name that do not themselves have plural marking are NNP, not NNPS, even if the head is NNPS.

E.g., "United States" is "United/ADJ+NNP States/PROPN+NNPS" in EWT and "United/VERB+NNP States/PROPN+NNPS" in GUM.

**Comment by amir-zeldes** [comment]
> E.g., "United States" is "United/ADJ+NNP States/PROPN+NNPS" in EWT and "United/VERB+NNP States/PROPN+NNPS" in GUM.

Agreed about the Number, but why is "United" ADJ in EWT? I thought this was consistent, aren't the States literally "United" with each other? If it means that they have undergone unionization, then shouldn't the lemma be "Unite"?

**Comment by amir-zeldes** [comment]
Yes, see my comment just now [here](https://github.com/UniversalDependencies/UD_English-EWT/issues/480#issuecomment-1847628011). ON has both tags for lowercased "united", though JJ is the majority tag. For the criteria in the link, the relative paraphrase decides it for VBN ("united people" can be "people who are united by ...")
